src/core/ext/transport/chttp2/server/chttp2_server.cc

```diff
@@ -674,13 +674,12 @@ grpc_error_handle Chttp2ServerListener::Create(
       auto string_address = grpc_sockaddr_to_uri(addr);
       if (!string_address.ok()) {
         return GRPC_ERROR_CREATE_FROM_CPP_STRING(
-            std::string(string_address.status().ToString()));
+            string_address.status().ToString().c_str());
       }
       listener->channelz_listen_socket_ =
           MakeRefCounted<channelz::ListenSocketNode>(
-              string_address.value().c_str(),
-              absl::StrFormat("chttp2 listener %s",
-                              string_address.value().c_str()));
+              string_address->c_str(),
+              absl::StrFormat("chttp2 listener %s", string_address->c_str()));
```

Comment:
You do not need to call `c_str()` here, because `absl::StrFormat()` accepts `std::string`.